### PR TITLE
feat(tools): setup TailwindCSS in ui-components

### PR DESCRIPTION
**Pull request by huyenltnguyen**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Ref #41580

This PR adds basic TailwindCSS setup to `ui-components`. Setup steps are:
1. Started with the installation guide (https://tailwindcss.com/docs/installation#installing-tailwind-css-as-a-post-css-plugin)
2. Installed the [postcss-import](https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports) plugin
3. Created a config file for Tailwind (https://tailwindcss.com/docs/installation#create-your-configuration-file)
4. Created `global.css` to include all the Tailwind import (https://tailwindcss.com/docs/installation#include-tailwind-in-your-css)
5. Tested the setup by adding a custom color in the config file and applying it to the Button component via a class name

### Screenshot

| Before | After |
| --- | --- |
| Screen Shot 2021-05-16 at 22 42 16 | Screen Shot 2021-05-16 at 22 43 20 |

### CODE REVIEW

File: tools/ui-components/src/button.css [hunk 1]
```diff
@@ -28,3 +30,7 @@
   font-size: 16px;
   padding: 12px 24px;
 }
+
+.fcc-style {
+  @apply bg-darkGreen;
+}
```
Comment (huyenltnguyen) [inline tools/ui-components/src/button.css:36]: This is just to validate that tailwind is working. The button does not match our styles and its implementation needs to be updated (IMO). I am planning to tear it down and rebuild it in a coming up PR as the pilot component.

File: tools/ui-components/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  purge: ['./src/**/*.html', './src/**/*.js'],
+  darkMode: false,
+  theme: {
+    colors: {
+      // Configure the color palette here
+      darkGreen: '#00471b'
+    }
```
Comment (huyenltnguyen) [inline tools/ui-components/tailwind.config.js:8]: I think we can start adding the color tokens in after this PR gets merged, and the task can be open for new contributors. The color list should contain [layout colors](https://design-style-guide.freecodecamp.org/#layout-colors) and [accent colors](https://design-style-guide.freecodecamp.org/#accent-colors).

File: tools/ui-components/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  variants: {
+    extend: {}
+  },
+  plugins: []
```
Comment (huyenltnguyen) [inline tools/ui-components/tailwind.config.js:13]: These are just placeholders that `npx tailwindcss init` added.

**Review by raisedadead — APPROVED** [review]

File: tools/ui-components/tailwind.config.js (hunk 3 above)
Comment (CRYONE7) [reply tools/ui-components/tailwind.config.js:13]: Hi I Am Crypto808...
Jsvsg
```suggestion
  variants: {
    extend: {}
  },
  plugins: []
[disputeresolution_belgium_en.pdf](https://github.com/freeCodeCamp/freeCodeCamp/files/6738296/disputeresolution_belgium_en.pdf)

```

**Pull request merged**